ReactiveCocoa/UIKit/UIScrollView.swift

```diff
@@ -31,4 +31,9 @@ extension Reactive where Base: UIScrollView {
 	public var maximumZoomScale: BindingTarget<CGFloat> {
 		return makeBindingTarget { $0.maximumZoomScale = $1 }
 	}
+
+	/// Sets whether the scroll view scrolls to the top when the menu is tapped.
+	public var scrollsToTop: BindingTarget<Bool> {
```

Comment:
It looks like this needs to be annotated to only be available on iOS (not tvOS).

https://travis-ci.org/ReactiveCocoa/ReactiveCocoa/jobs/244371741